### PR TITLE
Create ttt.js

### PR DESCRIPTION
**Pull request by KewlStatics**
Tic Tac Toe

### CODE REVIEW

**Comment by Spandamn** [comment]
Who wrote the code? There is no name, you cannot pass somebody else's code as yours.

**Comment by KewlStatics** [comment]
Its a code made by haze server! I helped in making it

**Comment by KewlStatics** [comment]
let me add the name then

**Comment by KewlStatics** [comment]
@Spandamn

File: chat-plugins/ttt.js [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+};
+
+exports.commands = {
+	ttt: 'tictactoe',
+	tictactoe: cmds,
+	tttend: 'endttt',
+	endttt: cmds.end,
+
+	//PM command override
+	pm: 'msg',
```
Comment (Spandamn) [inline chat-plugins/ttt.js]: What is this for?

Comment (KewlStatics) [inline chat-plugins/ttt.js]: ohh! Like it is when I Challenge you or some1 else for the game the challenges strike into the pm's

Comment (KewlStatics) [inline chat-plugins/ttt.js]: and when we end the game the commands dont work after the game

**Pull request merged**